llvm/lib/CodeGen/SelectionDAG/LegalizeVectorTypes.cpp

```diff
@@ -5982,6 +5985,89 @@ SDValue DAGTypeLegalizer::WidenVecRes_INSERT_VECTOR_ELT(SDNode *N) {
                      N->getOperand(1), N->getOperand(2));
 }
 
+static SDValue loadElement(SDValue LdOp, EVT FirstVT, EVT WidenVT,
```

Comment:
Document this. The name is misleading, no load is performed here.

We also have way too many places essentially repeating the same bit coercion dance

Reply:
Better name might be coerceLoadedValue?